common/api/ui-framework.api.md

```diff
@@ -2134,7 +2134,7 @@ export class FrameworkAccuDraw extends AccuDraw implements UserSettingsProvider
 export const FrameworkReducer: (state: import("./redux-ts").CombinedReducerState<{

```

Comment:
Please fix extracts by doing the following: (do not use rush extract for this)
1. cd to ui\framework 
2. npm run clean && npm run build && npm run extract-api
3. commit corrected md file

Reply:
done